### PR TITLE
#82 - Update instructions to match the UI of Raven Studio (RavenDB 5.1)

### PR DESCRIPTION
**Pull request by omerlagziel**
https://github.com/ravendb/bootcamp/issues/82

### CODE REVIEW

**Comment by arekpalinski** [comment]
Actually we have never updated our bootcamp to version 5.x (hence we still have just `v3.x` and `v4.x`) branches. @omerlagziel was it the only issue that you have noticed?

This change should be merged to `v5.x` branch (which doesn't exist yet) and then we could default to it when sending the bootcamp lessons via email

**Comment by omerlagziel** [comment]
@arekpalinski Just updated the branch with the other issues I noticed.

**Comment by arekpalinski** [comment]
Thanks!

File: src/unit3/lesson2/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Start Visual Studio and create a new `Console Application Project` named
 command:
 
 ```powershell
-Install-Package RavenDB.Client -Version 4.0.3
+Install-Package RavenDB.Client -Version 5.1.8
```
Comment (gregolsky) [inline src/unit3/lesson2/README.md]: Can you add `MinimumVersion` and `MaximumVersion` option here?
See docs on this https://docs.microsoft.com/en-us/powershell/module/packagemanagement/install-package?view=powershell-7.1

```suggestion
Install-Package RavenDB.Client -MinimumVersion 5.1.0 -MaximumVersion 5.9.9
```

Please verify it works properly

And in the other place as well please.

Comment (omerlagziel) [reply src/unit3/lesson2/README.md]: The `MinimumVersion` and `MaximumVersion` are not supported, as well as wildcards.
For .NET Core based projects `dotnet add package --version 5.*` can be used.

See: https://stackoverflow.com/questions/49795616/nuget-install-package-with-wildcard-version

Comment (gregolsky) [reply src/unit3/lesson2/README.md]: It's there in the docs - should be supported. What's your PS version?

Comment (gregolsky) [reply src/unit3/lesson2/README.md]: Ah, it says there on top it needs to be run within Package Manager Console in Visual Studio. Hmm

Comment (gregolsky) [reply src/unit3/lesson2/README.md]: Ok it doesn't work for me either.. Let's make it `5.2.0` and move along.

Comment (omerlagziel) [reply src/unit3/lesson2/README.md]: Done

**Pull request merged**